src/lib/php/common-sysconfig.php

```diff
@@ -795,3 +795,31 @@ function set_python_path()
   putenv("PYTHONPATH=/home/" . $SysConf['DIRECTORIES']['PROJECTUSER'] .
       "/pythondeps");
 }
+/**
+ * \brief Validation function get_system_load_average().
+ *
+ * Get no of cores using nproc command.
+ * Get load using sys_getloadavg
+ *
+ * \return button with different colors
+ */
+function get_system_load_average()
+{
+  // Get No.of cores
+  $cores = trim(shell_exec("nproc"));
+
+  // Get CPU load
+  $load = sys_getloadavg()[0];
```

Comment:
IMO, using the last 5 minute average will be much better than just 1 minute as load can fluctuate a lot when scanners are running on the server.